internal/test/wsu/wsu_test.go

```diff
@@ -98,32 +98,45 @@ func testAllVMs(t *testing.T) {
 func setupAndTest(t *testing.T, vmNum int) {
 	// Indicate that we can run the test suite on each node in parallel
 	t.Parallel()
-
+	vm := framework.WinVMs[vmNum]
 	// Run the WSU against the VM
-	wsuOut, err := runWSU(framework.WinVMs[vmNum])
-	require.NoError(t, err, "WSU playbook returned error: %s, with output: %s", err, wsuOut)
-
+	wsuOut, err := runWSU(vm)
+	wsuStringOutput := string(wsuOut)
+	require.NoError(t, err, "WSU playbook returned error: %s, with output: %s", err, wsuStringOutput)
+	// TODO: Think of a better way to refactor this function later to get just output that can be consumed by framework.
+	logFileName := t.Name() + "wsu.log"
+	externalIP := vm.GetCredentials().GetIPAddress()
+	nodeName, err := framework.GetNodeName(externalIP)
+	if err != nil {
+		// TODO: Change all fmt.Printfs to log.Printfs
```

Comment:
This is the first case of logging in this file. Please use `log` for consistency.